projects/managed-hubs/support.md

```diff
@@ -173,24 +174,62 @@ This process is carried out in an ongoing basis by the {term}`Support Stewards`.
 
    ```
    Hello { NAME }, thanks for sending us your request. We've concluded that
    this is not related to an outage or diminished service, so we'll incorporate
    this request into our team's work planning process. We'll send an update
    when we've got a plan for completing this request.
    ```
-5. **Prioritize the request**. Work with our team to decide how we should prioritize this request relative to the other work we need to do. Follow the [how to prioritize Change and Guidance Requests guide](support:prioritize-requests).
-6. **Resolve the request**. Our team should work together to prioritize and resolve this request according to its impact.
-7.  **Confirm resolution**. Once we have resolved a support request, send a message to the Community Representative to confirm that we believe it is resolved. In FreshDesk, mark the incident as {guilabel}`Resolved`.
-8.  **Close the request**. If the Community Representative confirms that their request has been fulfilled, or if we have not gotten a response after 48 working hours, consider this request closed. In FreshDesk, mark the incident as {guilabel}`Closed`.
+7. `Who: Engineering Manager (currently assigning reactive work) or someone delegated by Engineering Manager if he is not available`
+
+   **Prioritize the request**. Any non-`critical` issue should wait to be included in our sprints (on Wednesdays, every other week) to be worked out as part of the "planned" reactive work. Follow the [how to prioritize Change and Guidance Requests guide](support:prioritize-requests) to decide how we should prioritize this request relative to the other work we need to do. We should be fully transparent about the support queue to our Community Representatives if they ping us for updates.
+   
+   If there is any `critical` issue, we could assign people on the fly (during the sprint) to resolve them, but we should minimize that behavior (it should be exceptional cases).
+9. `Who: Support steward`
+
+   **Resolve the request**. When some engineer is assigned to a support-related GH issue in the context of a sprint, we move ahead with the investigation/resolution for one (1) sprint. If we failed to find a fix during that time, we communicate back that state in the Freshdesk ticket and resolve it.
+
+   Exceptional tickets might need more than one sprint. These tickets need to be explicitly approved as exceptions.
+11. `Who: Support steward`
+
+   **Confirm resolution**. Once we have resolved a support request, send a message to the Community Representative to confirm that we believe it is resolved. In FreshDesk, mark the incident as {guilabel}`Resolved`.
+11. **Who: Support steward**
+
+   **Close the request**. If the Community Representative confirms that their request has been fulfilled, or if we have not gotten a response after 48 working hours, consider this request closed. In FreshDesk, mark the incident as {guilabel}`Closed`.
```

Comment:
If possible we could add automation to transition "resolved" to "closed" after 48 working hours, but I'd say we are fine without it as well. I don't think we should add a manual task to transition resolved to closed.

I've seen both "Resolved" and "Closed" as ticket statuses both meaning that they are finalized, but that "Resolved" indicated that the ticket was finalized by taking successful action or similar, while "Closed" indicated that the ticket was finalized in some other way.

I don't think there is much importance for us to distinguish this, and I don't mind that we have different meanings.

```suggestion
11. **Who: Support steward**

   **Close the request**. If the Community Representative confirms that their request has been fulfilled, consider this request closed. In FreshDesk, mark the incident as {guilabel}`Closed`.
```

Reply:
> I don't think there is much importance for us to distinguish this, and I don't mind that we have different meanings.

I tend to agree given the current state of things.